games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
```

Comment:
Author is an object instead of just string. When using such file for building gallery there will be authors.json file available. But in case of companion file it will be nice to provide proper name.